incubator/kube-backup/templates/mysql.cronjob.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.mysql.enabled -}}
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: {{ template "fullname_mysql" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  concurrencyPolicy: {{ .Values.mysql.concurrencyPolicy }}
+# @TODO: Uncomment it when helm version > 2.3.1 would be used
```

Comment:
nope it should be > then 2.3.1 not >=
This version is not released yet. 
I just saw in master branch that they fixed issue.
So waiting new release